api-parent/core-api/src/main/java/com/sap/cloud/environment/servicebinding/api/DefaultServiceBinding.java

```diff
@@ -252,6 +248,14 @@ public interface TerminalBuilder
         @Nonnull
         TerminalBuilder withServiceIdentifier( @Nonnull final ServiceIdentifier serviceIdentifier );
 
+        /**
+         * Removes the {@link ServiceIdentifier} from the to-be-built {@link DefaultServiceBinding}.
+         *
+         * @return This {@link TerminalBuilder} instance.
+         */
+        @Nonnull
+        TerminalBuilder withoutServiceIdentifier();
```

Comment:
I'm not sure whether I like this new API...
What do you think about changing the `withServiceIdentifier(@Nonnull ...)` to `withServiceIdentifier(@Nullable ...)`?
(We could also do that for the remaining `with...` APIs, so that it stays more consistent overall.